### DISCUSSION TITLE
Bug: Cannot use double click to double tap in virtual device

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#13] [Genymotion Desktop] [answered]
If I click with mouse left button fast twice at one position on the virtual device, the virtual device only receives one tap. I'm using Ubuntu 20.04.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @UlyssesZh,

Thank you very much for your report!

This is actually a known bug with Genymotion Desktop and a bug request has already been filed. We don't have any ETA for a bug fix, however, but it will be released eventually.

Thank you for your patience!
Best regards,
Julien

**Reply by UlyssesZh** [reply]
Thank you for responding, but three years passed and I am still waiting for this seemingly small bug to be fixed.

**Comment by jmaneyrol69** [comment]
Hi @UlyssesZh ,
I am terribly sorry for the lack of updates.

Unfortunately there haven't been any updates to the bug requests on our end.
I have notified our dev team.

Thank you for your understanding.

**Comment by MuhammadAllam** [comment]
I'm experiencing the exact same issue on Windows as well, not only on Linux.

This problem is blocking basic interactions in many apps.
Could you please prioritize fixing it or suggest a workaround for Windows users too?

Thanks!

**Comment by muddinw** [comment]
Try new version of genymotion or another operating system
